src/span-data.ts

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+import {randomBytes} from 'crypto';
```

Comment:
It would probably be beneficial to try and use [`crypto.randomFillSync()`](https://nodejs.org/dist/latest-v8.x/docs/api/crypto.html#crypto_crypto_randomfillsync_buffer_offset_size) if it exists. It is ~30% faster last time I benchmarked it

Reply:
Done. Thanks for the tip.